src/fairchem/core/common/relaxation/ase_utils.py

```diff
@@ -203,14 +246,12 @@ def __init__(
             self.trainer.set_seed(seed)
 
         self.a2g = AtomsToGraphs(
-            max_neigh=max_neighbors,
-            radius=cutoff,
             r_energy=False,
             r_forces=False,
             r_distances=False,
-            r_edges=False,
```

Comment:
What was the reason to change these arguments? Because they don't matter since the model is constructing the graph?

I would say to keep `r_edges=False` if that is the case, otherwise no need to construct the pymatgen neighborlist.

Reply:
Taking it from the model now, if otf_graph is True.

This is not the cleanest solution, but I prefer leaving cleaning this up to another PR.